Podcast/DiscoverNewViewController.swift

```diff
@@ -0,0 +1,177 @@
+        searchBar.searchBar.placeholder = "Search"
+        navigationItem.titleView = searchBar.searchBar
+        searchBar.searchBar.sizeToFit()
+        searchBar.hidesNavigationBarDuringPresentation = false
+        searchBar.dimsBackgroundDuringPresentation = false
+        
+        tableView = UITableView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height - appDelegate.tabBarController.tabBarHeight), style: .grouped)
+        for (cls, identifier) in zip(sectionContentClasses, sectionContentIndentifiers) {
+            tableView.register(cls.self, forCellReuseIdentifier: identifier)
+        }
+        tableView.backgroundColor = .podcastGray
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.separatorStyle = .none
+        tableView.showsVerticalScrollIndicator = false
+        tableView.contentInset = UIEdgeInsets(top: -1, left: 0, bottom: 0, right: 0)
+        view.addSubview(tableView)
+        
+        // Populate with dummy data
+        let s = Series()
+        s.title = "Design Details"
+        s.nSubscribers = 832567
+        series = Array(repeating: s, count: 7)
+        tags = ["Education", "Politics", "Doggos", "Social Justice", "Design Thinking", "Science", "Mystery"]
+        let episode = Episode(id: 0)
+        episode.title = "Puppies Galore"
+        episode.series = s
+        episode.dateCreated = Date()
+        episode.smallArtworkImage = #imageLiteral(resourceName: "filler_image")
+        episode.descriptionText = "We talk lots about dogs and puppies and how cute they are and the different colors they come in and how fun they are."
+        episode.tags = ["Design", "Learning", "User Experience", "Technology", "Innovation", "Dogs"]
+        episodes = Array(repeating: episode, count: 5)
+    }
+    
+    //MARK: - TableView DataSource & Delegate
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        // don't know how to condense this using an array like the other functions
+        switch indexPath.section {
+        case 0:
+            let cell = tableView.dequeueReusableCell(withIdentifier: sectionContentIndentifiers[indexPath.section]) as! RecommendedTagsTableViewCell
+            cell.dataSource = self
+            cell.delegate = self
+            return cell
+        case 1:
+            let cell = tableView.dequeueReusableCell(withIdentifier: sectionContentIndentifiers[indexPath.section]) as! RecommendedSeriesTableViewCell
+            cell.dataSource = self
+            cell.delegate = self
+            return cell
+        case 2:
+            let cell = tableView.dequeueReusableCell(withIdentifier: sectionContentIndentifiers[indexPath.section]) as! RecommendedEpisodesOuterTableViewCell
+            cell.dataSource = self
+            cell.delegate = self
+            return cell
+        default:
+            return UITableViewCell()
```

Comment:
to avoid this do

guard let cell = tableView.dequeueReusableCell(withIdentifier: sectionContentIndentifiers[indexPath.section]) else { return UITableViewCell() } 

at the beginning of this function, then try to unwrap cell using if let and casting with as?